programs/management/commands/import-catalog-descriptions.py

```diff
@@ -314,18 +315,50 @@ def __get_catalog_entries(self):
                 and 'academicLevel' in result
                 and ('programTypeUndergrad' in result or 'programTypeGrad' in result)
             ):
+                self.catalog_entry_data_queue.put(result)
+
+        self.catalog_entry_data_progress = ChargingBar(
```

Comment:
Not particularly important, but would be nice to add `self.catalog_entry_data_progress = None` in the command handler method for consistency with our other progress bars